autorest/codegen/models/primitive_schemas.py

```diff
@@ -26,15 +26,13 @@ def _add_optional_and_default_value_template_representation(
     *,
     optional: bool = True,
     default_value_declaration: Optional[str] = None,
-    description: Optional[str] = None,
     **kwargs: Any
 ):
+    # for better display effect, use dafault value instead of var type
+    if default_value_declaration and default_value_declaration != "None":  # not doing None bc that's assumed
```

Comment:
I think the behavior I used to have here is wrong. We should have `None` if that's the default value, since I no longer believe it's assumed, and we want the default value to show up in the template regardless of whether it's `None` or not